index.html

```diff
@@ -32,7 +32,105 @@
             <h1 class="carousel-heading">Meet fun people</h1>
             <p class="carousel-paragraph">optional rating of other gamers</p>
           </div>
+          <a class="carousel-control-prev" href="#carouselControls" role="button" data-slide="prev">
+            <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+            <span class="sr-only">Previous</span>
+          </a>
+          <a class="carousel-control-next" href="#carouselControls" role="button" data-slide="next">
+            <span class="carousel-control-next-icon" aria-hidden="true"></span>
+            <span class="sr-only">Next</span>
+          </a>
         </div>
+      </header>
+
+      <nav class="navig container-fluid">
+        <ul class="lista row ">
+            <li
+                class="lista_home col-md-3">
+                <a
+                    href="#hero">HOME</a>
+            </li>
+            <li
+                class="lista_about col-md-3">
```

Comment:
should be list not `lista`

Reply:
it's not my contribution - Maks is responsible for this section and this is not a matter of my pull request